plugins/user-authenticators/saml2/src/org/apache/cloudstack/saml/SAML2AuthManagerImpl.java

```diff
@@ -119,12 +123,12 @@
     @Override
     public boolean start() {
         if (isSAMLPluginEnabled()) {
-            setup();
             s_logger.info("SAML auth plugin loaded");
+            return setup();
         } else {
```

Comment:
@bhaisaab I feel we can go with the if-else style because it anyhow is doing the same thing plus it give better code readability and logging. Your take?